Controller/Component/WizardComponent.php

```diff
@@ -510,6 +529,9 @@ protected function _validStep($step) {
  * @return void
  */
 	protected function _setCurrentStep($step) {
+		if (!in_array($step, $this->steps)) {
+			return;
+		}
```

Comment:
That would be the exact reason to leave comments in the code. Otherwise I would take out the code rather than just hiding it under private.